README.md

```diff
@@ -2,11 +2,88 @@
 Mock test suite for all language SDKs
 
 ## Requirements
-* Node 6.11 or newer
+* Node 6.11+
```

Comment:
Add additional details to let users know that Java is needed to run the mock server, but Node is only needed if they wish to act as an edit and re-bundle the code.